### DISCUSSION TITLE
Alternate mod is not mutually incopatible with Autoplay

### DISCUSSION THREAD

**Discussion by dekrain** [ppy/osu#16904] [Q&A] [answered]
OsuModAlternate is set to be incompatible with OsuModAutoplay, but the reverse is not true.

**Comment by bdach** [comment, marked as answer]
you're welcome to PR a change for this.

eventually the way to resolve this would be https://github.com/ppy/osu/issues/7155.

**Reply by dekrain** [reply]
Yeah, I think it's better to just rework the mod logic so this doesn't happen in the first place.
